react/features/base/react/components/native/LoadingIndicator.js

```diff
@@ -27,12 +27,25 @@ export default class LoadingIndicator extends Component<Props> {
      * @returns {ReactElement}
      */
     render() {
-        return (
+        let { size = 'large' } = this.props;
+
+        if (size === 'medium') {
+            size = 'large';
+        }
+
+        const props = {
+            animating: true,
+            color: ColorPalette.white,
+            ...this.props,
+            size
+        };
+
+        return (// $FlowFixMe
```

Comment:
what is this $FlowFixMe about ?

Reply:
This is related to the fact that size values accept medium. I think flow doesn't detect that we are actually overwriting this value and complains...